packetbeat/beater/packetbeat.go

```diff
@@ -26,48 +26,48 @@ import (
 )
 
 // Beater object. Contains all objects needed to run the beat
-type Packetbeat struct {
-	Config      config.Config
-	CmdLineArgs CmdLineArgs
-	Pub         *publish.PacketbeatPublisher
-	Sniff       *sniffer.SnifferSetup
+type packetbeat struct {
+	config      config.Config
```

Comment:
Well, packetbeat can be used as framework, not as a library. The New function for creating an instances is still available. The packetbeat internals are internal now, so no one messes with them when using packetbeat as framework.

Insteadt of using type beater.Packetbeat, one uses the type beat.Beater as returned by New method.